docs/basics/data-center-endpoints/data-center-endpoints.md

```diff
@@ -97,80 +94,84 @@ For details about setting up SSO for Sauce Labs, see [Setting Up Single Sign-On]
+| Headless US-East | `https://us-east-1.saucelabs.com/sso/metadata` | `https://us-east-1.saucelabs.com/sso/acs` |
 
 ## Changing Your Data Center
-If you have access to more than one DC, you can switch DCs by clicking the data center name in the upper-right corner. This drops down a list of options available to you.
+If you have access to more than one data center, you can switch data centers by clicking the data center name in the upper-right corner. This drops down a list of options available to you.
 
-Changing your DC changes what you can see and do in the Sauce Labs UI:
+Changing your data center changes what you can see and do in the Sauce Labs UI:
 
 * Live tests you run in the UI
-* Results for live tests and automated tests you run on the DC
-* Organization-wide concurrency settings specific to the DC
-* Teams associated with the DC
-* Sauce Connect Proxy Tunnels connected to the DC
-
-## Status Pages
-To check on the state of your data center, click the applicable link:
+* Results for live tests and automated tests you run on the data center
+* Organization-wide concurrency settings specific to the data center
+* Teams associated with the data center
+* Sauce Connect Proxy Tunnels connected to the data center
 
-US Data Center - [https://status.us-west-1.saucelabs.com/](https://status.us-west-1.saucelabs.com/)
-EU Data Center - [http://status.eu-central-1.saucelabs.com/](http://status.eu-central-1.saucelabs.com/)
-Headless Data Center (US-only) - [https://status.us-east-1.saucelabs.com/](https://status.us-east-1.saucelabs.com/)
+## Data Center Status
+To check on the status of your data center, see the [Sauce Labs System Status](https://status.saucelabs.com/) page.
 
 ### Maintenance Windows
 To view planned maintenance windows for your data center, see [Maintenance Windows](/dev/data-center-maint).
 
 ## Endpoints
-URLs vary based on the services you're using, your DC, and the types of tests you need to run.
+URLs vary based on the services you're using, your data center, and the types of tests you need to run.
 
-### US Data Center
+### US West Data Center
 | Name | Details |
 |---|---|
-| **US OnDemand Endpoints** |   |
+| **US West OnDemand Endpoints** |   |
 | Desktop Browser and Virtual Device Cloud | `https://ondemand.us-west-1.saucelabs.com/wd/hub` |
 | RDC on Sauce | `https://ondemand.us-west-1.saucelabs.com/` |
-| Legacy RDC/TestObject <small><span className="sauceGold">DEPRECATED</span></small>| `https://us1.appium.testobject.com/wd/hub` |
-| Team Management | `https://api.us-west-1.saucelabs.com/team-management/v1/` |
-| **US REST API** |   |
+| **US West REST API** |   |
 | Virtual Devices and Desktops | `https://api.us-west-1.saucelabs.com/v1` |
 | RDC on Sauce | `https://api.us-west-1.saucelabs.com/v1/rdc` |
-| Real Devices on TestObject <small><span className="sauceGold">DEPRECATED</span></small>| `https://app.testobject.com/api/rest/` |
```

Comment:
I might check with the TO Migration folks about deleting these TestObject endpoints. It is my understanding that we are leaving them in because there are some (rather big) customers who are still on the allowed list for using TO and we don't want to remove these refs until every customer who has been given extra time to migrate has done so. @linafarid  -- could you confirm/correct?

Reply:
Yeah - I think it is a reasonable request because of our state of technical end-of-life. I just wanted to make sure it doesn't go against the TO EOL team's efforts to preserve content for those few customers who are still in limbo, so I would be more comfortable if we give @linafarid (or maybe @skhg) a chance to respond.

Reply:
hi @lauramthomas and @sweeneyskirt-sl , thanks for bringing this up.
We do want to keep everything in the docs around TestObject as is with the addition of adding the "deprecated" label.
It would be better to keep it to stay consistent across all tech content areas. Thank you!